Jenkinsfile

```diff
@@ -8,6 +8,8 @@ def ssh_sh(String action) {
 }
 
 def formulae = [
+    "snips-platform-common.rb",
```

Comment:
Add a comment to keep it first so we don't forget that we need to build it first